docs/integrations/persisting-store-data.md

```diff
@@ -571,3 +571,34 @@ export const useBearStore = create<MyState>()(
   )
 )
 ```
+
+### How do I use deal with Map/Set with the persist api
```

Comment:
I'm not completely sure how to understand it, but I think it will be more readable with a shorter question:

```suggestion
### How do I use it with Map and Set?
```

We implicitly use the `persist` API on this doc page, so I omitted it.